packages/astro/src/runtime.ts

```diff
@@ -439,52 +431,31 @@ async function createSnowpack(astroConfig: AstroConfig, options: CreateSnowpackO
 /** Core Astro runtime */
 export async function createRuntime(astroConfig: AstroConfig, { mode, logging }: RuntimeOptions): Promise<AstroRuntime> {
   const timer: Record<string, number> = {};
-  const resolvePackageUrl = async (pkgName: string) => frontendSnowpack.getUrlForPackage(pkgName);
+  const resolvePackageUrl = async (pkgName: string) => snowpack.getUrlForPackage(pkgName);
 
   timer.backend = performance.now();
   const {
-    snowpack: backendSnowpack,
-    snowpackRuntime: backendSnowpackRuntime,
-    snowpackConfig: backendSnowpackConfig,
+    snowpack,
+    snowpackRuntime,
+    snowpackConfig,
   } = await createSnowpack(astroConfig, {
-    target: 'backend',
-    env: {
-      astro: true,
-    },
     mode,
     resolvePackageUrl,
   });
-  debug(logging, 'core', `backend snowpack created [${stopTimer(timer.backend)}]`);
-
-  timer.frontend = performance.now();
-  const {
-    snowpack: frontendSnowpack,
-    snowpackRuntime: frontendSnowpackRuntime,
-    snowpackConfig: frontendSnowpackConfig,
-  } = await createSnowpack(astroConfig, {
-    target: 'frontend',
-    env: {
-      astro: false,
-    },
-    mode,
-  });
-  debug(logging, 'core', `frontend snowpack created [${stopTimer(timer.frontend)}]`);
+  debug(logging, 'core', `snowpack created [${stopTimer(timer.backend)}]`);
 
   const runtimeConfig: RuntimeConfig = {
     astroConfig,
     logging,
     mode,
-    backendSnowpack,
-    backendSnowpackRuntime,
-    backendSnowpackConfig,
-    frontendSnowpack,
-    frontendSnowpackRuntime,
-    frontendSnowpackConfig,
+    snowpack,
+    snowpackRuntime,
+    snowpackConfig,
   };
 
   return {
     runtimeConfig,
     load: load.bind(null, runtimeConfig),
-    shutdown: () => Promise.all([backendSnowpack.shutdown(), frontendSnowpack.shutdown()]).then(() => void 0),
+    shutdown: () => Promise.all([snowpack.shutdown()]).then(() => void 0),
```

Comment:
Maybe remove the `Promise.all()` ? Unless we need to add more cleanup relatively soon

Reply:
oh yeah good point